### PR TITLE
108 news listing page

### PR DESCRIPTION
**Pull request by levmyshkin**
Add styles for News, Events, Blog  Posts content types

### CODE REVIEW

File: web/themes/custom/drupalam/drupalam.theme [hunk 1]
```diff
@@ -121,9 +121,8 @@ function drupalam_preprocess_image_formatter(&$vars) {
 }
 
 function drupalam_preprocess_node(&$variables) {
-
-  // Add comment count
-  if ($variables['node']->getType() == 'blog_post') {
+  // Add comment count.
+  if (in_array($variables['node']->getType(), ['blog_post', 'news'])) {
     $variables['comment_count'] = $variables['node']->get('field_blog_comments')->comment_count;
```
Comment (leo72) [inline web/themes/custom/drupalam/drupalam.theme:126]: Does News CT have `field_blog_comments `?

File: web/themes/custom/drupalam/js/blog-page.js [hunk 2]
```diff
@@ -34,12 +34,4 @@
     })
   }
 
-  function MasonryGrid() {
```
Comment (leo72) [inline web/themes/custom/drupalam/js/blog-page.js:37]: Why we remove  `MasonryGrid()` function?

**Review by leo72 — CHANGES_REQUESTED** [review]
@levmyshkin , please see my questions. I didn't review the final look, but CSS and TWIG look correct. Also I didn't review configs.

File: web/themes/custom/drupalam/drupalam.theme (hunk 1 above)
Comment (levmyshkin) [reply web/themes/custom/drupalam/drupalam.theme:126]: Yes, I resued field_blog_comments field and styles for News.

File: web/themes/custom/drupalam/js/blog-page.js (hunk 2 above)
Comment (levmyshkin) [reply web/themes/custom/drupalam/js/blog-page.js:37]: I can't remember. Restored masonry grid on Blog posts listing page.

Comment (leo72) [reply web/themes/custom/drupalam/js/blog-page.js:37]: I don't see it restored :-(

File: web/themes/custom/drupalam/drupalam.theme (hunk 1 above)
Comment (leo72) [reply web/themes/custom/drupalam/drupalam.theme:126]: Well, not the best solution, but let it be that way for now.

**Review by leo72 — CHANGES_REQUESTED** [review]
Something wrong with pull request, I still see `MasonryGrid()` as removed.

**Pull request merged**